src/NovelRT.Interop/Graphics/NrtTextRect.cpp

```diff
@@ -135,7 +135,14 @@ extern "C" {
     }
 
     TextRect* textRectPtr = reinterpret_cast<TextRect*>(rect);
-    textRectPtr->setText(std::string(inputText));
+    try
+    {
+      textRectPtr->setText(std::string(inputText));
+    }
+    catch (const Exceptions::CharacterNotFoundException)
+    { // todo: handle error message
```

Comment:
This looks like a custom exception. Do we have an internal function for the error message?